experimental/LieAlgebras/src/WeylGroup.jl

```diff
@@ -269,7 +261,7 @@ function Base.:(^)(x::WeylGroupElem, n::Int)
 end
 
 @doc raw"""
-    Base.:(<)(x::WeylGroupElem, y::WeylGroupElem)
+    <(x::WeylGroupElem, y::WeylGroupElem) -> Bool
 
 Returns whether `x` is smaller than `y` with respect to the Bruhat order.
```

Comment:
```suggestion
Returns whether `x` is smaller than `y` with respect to the Bruhat order,
i.e., whether some (not necessarily connected) subexpression of a reduced
decomposition of `y`, is a reduced decomposition of `x`.
```
Move some comment from below to the docstring